packages/frontend/src/features/metricFlow/utils/convertMetricFlowQueryResultsToResultsData.ts

```diff
@@ -46,19 +51,14 @@ export default function convertMetricFlowQueryResultsToResultsData(
                 };
             }, {}),
         ),
-        fields: {},
-    };
-
-    return {
-        resultsData,
-        fieldsMap: [...dimensionsInSchema, ...metricsInSchema].reduce<
```

Comment:
the `fieldsMap` information is inside `resultsData` so I simplified the returned value.